rpc_test.go

```diff
@@ -116,14 +116,17 @@ func TestUpdate(t *testing.T) {
 		t.Error("Expected: error for a dummy request")
 	}
 
+	// TODO fix and send valid row update
```

Comment:
This is such a complex and big change. I'd like to see tests added for this update. For example, what's the expected tableUpdates generated by update() to call handler.Update, test if the result is equal to the expected.